content/en/blog/2023/php-auto-instrumentation/index.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Opentelemetry PHP Auto-Instrumentation
+linkTitle: PHP Auto-Instrumentation
+date: 2023-03-15
+author: '[Przemek Delewski](https://github.com/pdelewski/) (Sumo Logic)'
+---
+
+## Introduction
+
+Automatic Instrumentation is a process of adding tracing capabilities into user
+application without modyfing it's source code. There are several techniques to
+do that, but all of them more or less work in the same way by injecting
+additional code into original one during compile time, link time, run-time or by
+extending the operating system in case of [ebpf](https://ebpf.io/). This
+blogpost presents method used by Opentelemetry PHP auto-instrumentation.
```

Comment:
Suggestion: Add a link to the auto instrumentation repo?

Reply:
@svrnm  Term "Opentelemetry PHP auto-instrumentation" here is used as an umbrella for whole auto-instrumentation work and at least for 2 repos, so I would leave it as it is if you don't mind.

Reply:
ACK